content/millennium/r4/financial/charge-item.md

```diff
@@ -0,0 +1,111 @@
+---
+title: ChargeItem | R4 API
+---
+
+# ChargeItem
+
+* TOC
+{:toc}
+
+## Overview
+
+The resource ChargeItem describes the provision of healthcare provider products for a certain patient, therefore referring not only to the product, but containing in addition details of the provision, like date, time, amounts and participating organizations and persons.
+Main Usage of the ChargeItem is to enable the billing process and internal cost allocation.
```

Comment:
Let's reword to "The main usage..."

Reply:
Updated [here|https://github.com/cerner/fhir.cerner.com/pull/605/commits/64ddf564c76cd6938adb0bdff00feda5cff09c86#diff-16d4f8f74626c62d57a8b71d6be699f60e72fffed8636bc32e6654d4252b3ebbR13]

Reply:
Thanks!